<!doctype html>
<html lang="en">
<head>
  <meta charset="utf-8">
  <title>LVZ Retreat, 2017</title>
  <meta name="description" content="">
  <meta name="author" content="Mukarram Hossain">
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
  <link rel="stylesheet" href="libraries/frameworks/revealjs/css/reveal.min.css">
  <link rel="stylesheet" href="libraries/frameworks/revealjs/css/theme/default.css" id="theme">
  <link rel="stylesheet" href="libraries/highlighters/highlight.js/css/zenburn.css" id="theme">
  <!--[if lt IE 9]>
  <script src="lib/js/html5shiv.js"></script>
  <![endif]-->  <link rel="stylesheet" href = "assets/css/ribbons.css">

</head>
<body>
  <div class="reveal">
    <div class="slides">
      <section class='' data-state='' id='slide-1'>
  <h2>Subtype classification using lossless compression techniques</h2>
  <p><br></br>
<strong>Mukarram Hossain</strong>
<br></br>
Department of Veterinary Medicine<br>
University of Cambridge
<br></br>
LVZ Retreat, 2017 
<br></br>
<img src="assets/img/uc-colour.png" width="30%" style="border: 0px">&nbsp;</p>

</section>
<section class='' data-state='' id='slide-2'>
  <h3>Lossless compression</h3>
  <p><br><br></br></br>
<code>Lossless compression is a class of data compression algorithms that allows the original data to be perfectly reconstructed from the compressed data.</code></p>

</section>
<section class='' data-state='' id='slide-3'>
  <h3>Subtype classification</h3>
  <ul>
<li>Viruses are often grouped into subtypes.</li>
<li>Subtypes have wide implications on the following studies of viruses:

<ul>
<li>clinical</li>
<li>epidemiological</li>
<li>structural</li>
<li>functional</li>
</ul></li>
<li>Existing classification techniques mostly rely on alignments followed by phylogenetic and/or statistical algorithms.</li>
</ul>

</section>
<section class='' data-state='' id='slide-4'>
  <h3>Alignment uncertainty - 1</h3>
  <p><img src="assets/img/ha.sp.png" width="65%" style="border: 0px">&nbsp;</p>

</section>
<section class='' data-state='' id='slide-5'>
  <h3>Alignment uncertainty - 2</h3>
  <p><img src="assets/img/denv.sp.png" width="100%" style="border: 0px">&nbsp;</p>

</section>
<section class='' data-state='' id='slide-6'>
  <h3>Alignment-free classification</h3>
  <ul>
<li>Lossless compression techniques have shown promising results for biological sequence classification:

<ul>
<li>Begleiter et al. (2004)</li>
<li>Ferragina et al. (2007)</li>
</ul></li>
</ul>

</section>
<section class='' data-state='' id='slide-7'>
  
  <p><br></br>
<img src="assets/img/comet_title.png" width="100%" style="border: 0px">&nbsp;</p>

</section>
<section class='' data-state='' id='slide-8'>
  <h3>COMET</h3>
  <ul>
<li>COMET is an ultrafast alignment free subtyping tool</li>
<li>Uses <strong>P</strong>rediction by <strong>P</strong>artial <strong>M</strong>atching (PPM)</li>
<li>Initially designed for HIV-1</li>
<li>COMET was tested on both synthetic (1090698) and clinical (10625) HIV datasets</li>
<li>Sensitivity and specificity were comparable to or higher than:

<ul>
<li>REGA (de Oliveira <em>et al.</em>, 2005) and </li>
<li>SCUEAL (Pond <em>et al.</em>, 2009)</li>
</ul></li>
<li>Detected and identified new recombinant forms</li>
</ul>

</section>
<section class='' data-state='' id='slide-9'>
  <h3>PPM - model</h3>
  <p><img src="assets/img/ppm.png" width="80%" style="border: 0px">&nbsp;</p>

</section>
<section class='' data-state='' id='slide-10'>
  <h3>PPM - likelihoods</h3>
  <p><img src="assets/img/ppm_prob.png" width="100%" style="border: 0px">&nbsp;</p>

</section>
<section class='' data-state='' id='slide-11'>
  <h3>COMET algorithm</h3>
  <ul>
<li>Builds variable-order Markov models for each reference sequence</li>
<li>Given a query, COMET calculates log likelihood of observing a base at each positions</li>
<li>This results in a matrix of likelihood values</li>
<li>Subtype call is done using a decision tree</li>
</ul>

</section>
<section class='' data-state='' id='slide-12'>
  <h3>The decision tree</h3>
  <p><img src="assets/img/decision_tree.png" width="600px" height="650px" style="border: 0px">&nbsp;</p>

</section>
<section class='' data-state='' id='slide-13'>
  <h3>COMET - availability</h3>
  <ul>
<li>COMET is free to use via an <a href="https://comet.lih.lu/">online</a> interface</li>
<li>Stand-alone Java jar file can be requested from the author via <a href="mailto:daniel.struck@lih.lu">daniel.struck@lih.lu</a></li>
</ul>

</section>
<section class='' data-state='' id='slide-14'>
  <h3>pyCOMET</h3>
  <p><img src="assets/img/pycomet.png" width="950px" height="650px" style="border: 0px">&nbsp;</p>

</section>
<section class='' data-state='' id='slide-15'>
  <h3>pyCOMET - results</h3>
  <p><img src="assets/img/pycomet_results.png" width="100%" style="border: 0px">&nbsp;</p>

</section>
<section class='' data-state='' id='slide-16'>
  <h3>Classification using Neural networks</h3>
  <ul>
<li><em>Neural networks</em> are computational system mimicking biological brain</li>
<li>Consists of a cluster of neural unit organised in layers</li>
</ul>

<p><br></br>
<img src="assets/img/ann.jpg" width="40%" style="border: 0px">&nbsp;</p>

</section>
<section class='' data-state='' id='slide-17'>
  <h3>Classification using RNN</h3>
  <ul>
<li>Recurrent neural networks are networks with connections so that information can persist</li>
</ul>

<p><br></br>
<img src="assets/img/rnn.png" width="60%" style="border: 0px">&nbsp;</p>

</section>
<section class='' data-state='' id='slide-18'>
  <h3>Classification using lossless compression</h3>
  <ul>
<li>&quot;<em>More accurate and more reliable</em>&quot; and &quot;<em>faster</em>&quot; than current best methods.</li>
<li>Robust, capable of identifying rare and novel recombinant forms.</li>
<li>Results are consistent on short segments as well as whole genomes.</li>
<li>Do not rely on alignments hence not biased by alignment uncertainty.
<br></br></li>
<li>Depends solely on the availability of complete reference sequence set</li>
</ul>

</section>
<section class='' data-state='' id='slide-19'>
  <h2>Plans for 2017</h2>
  
</section>
<section class='' data-state='' id='slide-20'>
  
  <p><img src="assets/img/grad.png" width="60%" style="border: 0px">&nbsp;</p>

</section>
<section class='' data-state='' id='slide-21'>
  
  <p><br></br>
<img src="assets/img/questions.jpg" width="600px" class="centred" style="margin: 10px 10px" /></p>

</section>
    </div>
  </div>
</body>
  <script src="libraries/frameworks/revealjs/lib/js/head.min.js"></script>
  <script src="libraries/frameworks/revealjs/js/reveal.min.js"></script>
  <script>
  // Full list of configuration options available here:
  // https://github.com/hakimel/reveal.js#configuration
  Reveal.initialize({
    controls: true,
    progress: true,
    history: true,
    center: true,
    theme: Reveal.getQueryHash().theme || 'default', 
    transition: Reveal.getQueryHash().transition || 'default', 
    dependencies: [
    // Cross-browser shim that fully implements classList -
    // https://github.com/eligrey/classList.js/
      { src: 'libraries/frameworks/revealjs/lib/js/classList.js', condition: function() { return !document.body.classList;}},
      // Zoom in and out with Alt+click
      { src: 'libraries/frameworks/revealjs/plugin/zoom-js/zoom.js', async: true, condition: function() { return !!document.body.classList; } },
      // Speaker notes
      { src: 'libraries/frameworks/revealjs/plugin/notes/notes.js', async: true, condition: function() { return !!document.body.classList; } },
      // Remote control your reveal.js presentation using a touch device
      //{ src: 'libraries/frameworks/revealjs/plugin/remotes/remotes.js', async: true, condition: function() { return !!document.body.classList; } }
      ]
  });
  </script>  <!-- LOAD HIGHLIGHTER JS FILES -->
<script src="libraries/highlighters/highlight.js/highlight.pack.js"></script>
<script>hljs.initHighlightingOnLoad();</script>
<!-- DONE LOADING HIGHLIGHTER JS FILES -->
 

</html>